packages/flutter/lib/src/material/elevated_button.dart

```diff
@@ -222,8 +222,9 @@ class ElevatedButton extends ButtonStyleButton {
   /// * `shadowColor` - Theme.shadowColor
   /// * `elevation`
   ///   * disabled - 0
-  ///   * hovered or focused - 2
-  ///   * pressed - 6
+  ///   * default - 2
```

Comment:
Is this a change in defaults, or was the previous comment incorrect?

Reply:
The previous comment was incorrect.